Workiva/anaplan-java-client
# STUD-804: Add Workiva CI

## Ticket

https://jira.atl.workiva.net/browse/STUD-804

Sets up Basic Workiva CI processes
- Dockerfile
- skynet.yaml
- dependabot
- aviary

## QA
- [] Build artifacts contain: 
    - [] anaplan-connect-[VERSION]-SNAPSHOT-javadoc.jar
    - [] anaplan-connect-[VERSION]-SNAPSHOT.jar (jar to satisfy RM's audit requirements)
- [] Package `anaplan-connect` appears on `maven-dev-internal` in Artifactory 
- [] Skynet results show unit tests are running and passing
- [] Sample regression chain passes with the new artifact
     - TODO - locate regression chain
